epsagon/events/dbapi.py

```diff
@@ -68,7 +68,6 @@ def __init__(
         """
 
         super(DBAPIEvent, self).__init__(start_time)
-
         self.event_id = 'dbapi-{}'.format(str(uuid4()))
 
         # in case of pg instrumentation we extract data from the dsn property
```

Comment:
those lines might fail too potentially (e.g. if dsn doesn't have a 'dbname' key). Consider printing a debug print before with the connection/dsn/query